src/generators/AbstractRenderer.ts

```diff
@@ -70,7 +70,7 @@ export abstract class AbstractRenderer<
           options, 
           model: this.model, 
           inputModel: this.inputModel
-        });
+        }) || content;
```

Comment:
What about if someone want to disable the given preset chain and return the empty string, what then?

Reply:
That is the question isn't it (https://github.com/asyncapi/modelina/pull/352#issuecomment-912511672) 😄

Reply:
Question 😄  I think we should only do fallback for `null` and `undefined`, empty string should remove preset chain.

Reply:
@magicmatatjahu are you sure we even need to handle this at all? 🤔

Reply:
Yes :)

Reply:
@magicmatatjahu now that I think about it, null or undefined, should never be possible 🤔 https://github.com/asyncapi/modelina/blob/253462b302e67cd1f3021059f0c36569cdd4155e/src/models/Preset.ts#L31

Reply:
In TS, but in JS you can return the null or undefined. I think that we should check in the function https://github.com/asyncapi/modelina/blob/2f4f8631801f92ade0324f2178180237cd4edf41/src/generators/AbstractRenderer.ts#L59 if given output is string or not, and then save or not the result.

Reply:
@magicmatatjahu right. 

I can't figure out if it makes more sense that `null` or `undefined` remove preset, or empty string 🤔 I am kinda leaning towards `null` or `undefined`

Reply:
... I dont know 😆

Reply:
Consider that presets act like middlewars in http frameworks. If a given middleware returns value without running the `next` function, then you break the middleware chain. It should be the same here - an empty string should also break the chain of presets. `null`, `undefined` and other non-string values we should treat as empty string.

Reply:
So the following is what you are proposing, in terms of return values from presets?
- empty string: default back to previous rendered preset.
- null, undefined and other non-string values: render preset as an empty string;

Reply:
- null, undefined and other non-string values: render preset as an empty string.
- empty string should be also saved to the content as in present solution.

The 1st point is a part of the 2nd point.

Reply:
I think that here we have problem with direction of the presets flow. At the we have only a pure, one direction flow - from first to the last. Maybe we should consider make the preset flow as bi-directiional as described [in this issue](https://github.com/asyncapi/modelina/issues/125).

Why is this a problem? Because if someone want to disable the given preset must add additional custom preset to the end of the flow and return the empty string. Of course there is also another (more needed) use cases for the bi-directional presets Maybe we should switch to the logic like in native middleware with `next` function? @jonaslagoni What do you think?

Reply:
I think that this PR is not about bi-directional presets 😄

I have no use-case where I see this as being useful at the moment, so I am just trying to address the associated issue 😄